haskell/test/Test/SegCFMM/Types.hs

```diff
@@ -11,7 +12,14 @@ import Test.HUnit (Assertion)
 
 import Michelson.Typed (Contract)
 
+import SegCFMM.Types (Storage)
 import Test.SegCFMM.Contract
+import Test.SegCFMM.Storage
 
-unit_TypesMatch :: Assertion
-unit_TypesMatch = evaluateNF_ @(Contract _ _) segCFMMContractLigo
+unit_ContractTypesMatch :: Assertion
+unit_ContractTypesMatch = evaluateNF_ @(Contract _ _) segCFMMContract
+
+unit_StorageTypesMatch :: Assertion
+unit_StorageTypesMatch = do
+  let !(_ :: Storage) = defaultStorage
+  pure ()
```

Comment:
@rinn7e I suppose that this is ok, although it doesn't really add anything in term of checks.

IMO once we are using `defaultStorage` in our actual tests we can get rid of this, so for now I'd say to keep this around, but also please add a TODO to remove this once it's no longer useful.

P.s. I suppose that a cleaner way to define this might be:
```suggestion
unit_StorageTypesMatch :: Assertion
unit_StorageTypesMatch = void $ pure (defaultStorage :: Storage)
```

Reply:
Thanks Pasquale, I updated this.